types/types.go

```diff
@@ -184,6 +184,7 @@ type NodeConfig struct {
 	WaitFor              []string   `json:"wait-for,omitempty"`
 	DNS                  *DNSConfig `json:"dns,omitempty"`
 	IsRootNamespaceBased bool
+	SkipUniquenessCheck  bool
```

Comment:
can we add a comment here when this might be set up to true with a ref to ext container

Reply:
done. please check